WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/PointOfSaleItemListErrorView.swift

```diff
@@ -11,11 +11,7 @@ struct PointOfSaleItemListErrorView: View {
 
     var body: some View {
         VStack(alignment: .center, spacing: PointOfSaleItemListErrorLayout.headerSpacing) {
-            Text(Localization.productTitle)
-                .frame(maxWidth: .infinity, alignment: .leading)
-                .padding(.vertical, PointOfSaleItemListErrorLayout.headerPadding)
-                .font(Font.posTitle)
-                .foregroundColor(Color.posPrimaryTexti3)
+            POSHeaderTitleView()
```

Comment:
```suggestion
            POSHeaderTitleView(foregroundColor: .posIconGrayi3)
```

Reply:
This seems to have changed as well since it was screenshotted 2 weeks ago 😭 [here](https://github.com/woocommerce/woocommerce-ios/issues/13480) despite the designs being "locked down". Updated here: https://github.com/woocommerce/woocommerce-ios/pull/13585/commits/d596f20324a3a69bc897e2684bbf57ce1e047a7d